api/events.go

```diff
@@ -115,6 +120,17 @@ func (eh *eventsHandler) Handle(e *cluster.Event) error {
 	data = data[:len(data)-1]
 	data = append(data, []byte(node)...)
```

Comment:
I'm not sure about this myself, but my guess is that this is related to backward compatibility. It's been around long enough that it should be removable now though. I can address that later.